docs/hiring-guide/community-spotlight/interview-celeste-horgan.rst

```diff
@@ -0,0 +1,85 @@
+    enough English to say, "No, that's not what I meant at all." So it's a It's a high level conversation. 
+    
+    And commercetools is a verydiverse company. For most of the people that I work with English is not their first language. It might
+    not be their secondlanguage. It's sometimes their third, fourth, or fifth language; because that's how Europe works. So when I
+    work with developers, if they're writing the first draft of something in particular, It takes a lot more work to get it to the
+    state that I would be comfortable releasing it in. And I have to be a lot more forward in how I speak to and question about a
+    feature, because I know that they might not be able to correct me if I misinterpret something that they say. So there's a lot more
+    responsibility on me  on. And I would actually say for anybody else that's looking to move abroad that there's a lot of demand,
+    particularly in Berlin. It's not a bad place to be a technical writer, because there's extra value on you. You're not only a
+    technical writer, not only the person taking care of documentation that developers don't want to write; you're the person doing
+    that with the authoritativevoice of a native English speaker.
+
+**What advice would you give to new or aspiring tech writers in the community?**
+
+    The thing that I would say to a new writer in the community or to any new writer in general is to not to ignore your tech stack
+    and your user experience. I meet a lot of writers when I'm interviewing, and all they want to do is write documentation. And I
+    admire that focus. That's not me. That is not how I approach my job. And I admire people who just want to sit down and write, and
+    they're super happy about it. But I think documentation is so overlooked in so many companies. It is low priority for a lot of
+    product teams. It's not something a lot of people like to do. But it is a really key component of a developer's or a customer's
+    experience with the product, because they come to it when they're stuck. They come to it when they're at the low point in their
+    journey with the product, and it's your job to take them higher. And it's not just the content that matters in that regards. It's
+    also how people perceive things visually that has an effect on how they feel about things. So as much as we would like to believe
+    that good content saves the day every single time, we have in our minds an idea of what a professional looking layout looks like.
+    And if you're displaying your documentation on something that isn't a professional looking layout, that changes people's
+    perception of what they're reading. It changes the emotions around it. And they may not take what you're writing as seriously. 
+
+    And I would say don't ignore the tech stack, because if you want to foster collaboration, you need to have a tech stack that other
```

Comment:
There's a ton of `And`'s in this response (over 10!) -- we should edit these out.

Reply:
I've fixed it up, and I think it reads better. Let me know if I should do a harder edit.